packages/core/components/FileList/index.tsx

```diff
@@ -131,6 +131,8 @@ export default function FileList(props: FileListProps) {
 
     // Get a count of all files in the FileList, but don't wait on it
     React.useEffect(() => {
+        // Show loading state in UI instead of stale count from previously rendered query results
+        setTotalCount(null);
```

Comment:
The comment to `setTotalCount(null)` doesnt seem easy to interpret to me.

Reply:
updated in [be28c24](https://github.com/AllenInstitute/biofile-finder/pull/569/commits/be28c24513166bd9e1749a56f180ee0a0b04d217)